src/ReviewService.Abstractions/ReviewService.Extensions.cs

```diff
@@ -65,4 +65,24 @@ public static async Task TrackSecondaryActionCompleted<TReviewSettings>(this IRe
 			reviewSettings => reviewSettings with { SecondaryActionCompletedCount = reviewSettings.SecondaryActionCompletedCount + 1 }
 		);
 	}
+
+	/// <summary>
+	/// Tracks that a review was requested.
+	/// </summary>
+	/// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+	/// <param name="reviewService"><see cref="IReviewService{TReviewSettings}"/>.</param>
+	/// <param name="ct">The cancellation token.</param>
+	/// <returns><see cref="Task"/>.</returns>
+	public static async Task TrackReviewRequested<TReviewSettings>(this IReviewService<TReviewSettings> reviewService, CancellationToken ct)
```

Comment:
This should be not be public because the app doesn't actually know when the review was prompted.

The settings need to be updated by the ReviewService directly.